app/assets/v2/js/pages/new_bounty.js

```diff
@@ -437,7 +437,7 @@ Vue.mixin({
           web3.eth.sendTransaction({
             to: toAddress,
             from: selectedAccount,
-            value: web3.utils.toWei(String(vm.totalAmount.totalFee), 'ether')
+            value: web3.utils.toWei(new web3.utils.BN(BigInt(vm.totalAmount.totalFee.toFixed(18) * 1.0 * Math.pow(10, 18))).toString(), 'wei')
```

Comment:
We shouldn't need the `* 1.0` part, and could we get away with not using web3 altogether?

`BigInt(vm.totalAmount.totalFee.toFixed(18) * Math.pow(10, 18)).toString()`

Should give us the same result?

Reply:
but with BN I believe is a more secure way about gettting what web3 is waiting for, not 100% sure.